rind/main.go

```diff
@@ -6,10 +6,23 @@ import (
 	"net/http"
 
 	"github.com/owlwalks/rind"
+	"flag"
+	"os"
+	"github.com/golang/glog"
 )
 
+var rwDirPath   = flag.String("rwdir","/var/dns","dns storage dir")
+var listenIP    = flag.String("listenip", "8.8.8.8", "dns forward ip")
+var listenPort  = flag.Int("listenport", 53, "dns forward port")
+
 func main() {
-	dns := rind.Start("", []net.UDPAddr{{IP: net.IP{1, 1, 1, 1}, Port: 53}})
+	flag.Parse()
+	if err := ensureDir(*rwDirPath); err != nil {
```

Comment:
```suggestion
	if err := os.MkdirAll(*rwDirPath); err != nil {
	    
	}
```

Reply:
could you change to os.MkDirAll, in case we want to put them in non-existed subdir